notification/src/api/notification/notification.controller.js

```diff
@@ -1,6 +1,7 @@
 const _ = require('lodash')
 const httpUtils = require('../../utils/commons')
 const ctp = require('../../utils/ctp')
+const { hasValidHMACSignature } = require('../../utils/hmacValidator')
```

Comment:
```suggestion
const { hasValidHmacSignature } = require('../../utils/hmacValidator')
```

And also at other places.

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/273/commits/3982a9e46f010cbce494856967abc4e47a6f7070